evaluation.py

```diff
@@ -12,47 +12,66 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import go
+"""Evulation plays games between two opponents."""
+
 import time
-import numpy as np
 
-from dual_net import DualNetwork
 from gtp_wrapper import MCTSPlayer
 
 
 def play_match(black_net, white_net, games, readouts, verbosity):
+    """Plays matches between two players.
+
+    black_net: Instance of minigo.DualNetwork, a wrapper around a tensorflow
+        convolutional network.
+    white_net: Instance of the minigo.DualNetwork.
+    games: number of games to play. We play all the games at the same time.
```

Comment:
Now that we have vlosses, it makes sense to refactor this whole thing to just play a single game instead of the root-level parallelism.  it'll probably simplify the code a lot.

Reply:
Added a TODO